packages/context/src/index.ts

```diff
@@ -19,12 +19,19 @@ export {
   getDeepProperty,
 } from './value-promise';
 
-export {Binding, BindingScope, BindingType, TagMap} from './binding';
+export {
+  Binding,
+  BindingScope,
+  BindingType,
+  TagMap,
+  BindingTemplate,
+} from './binding';
 
 export {Context} from './context';
 export {BindingKey, BindingAddress} from './binding-key';
 export {ResolutionSession} from './resolution-session';
 export {inject, Setter, Getter, Injection, InjectionMetadata} from './inject';
+export * from './binding-decorator';
```

Comment:
Let's try to keep these exports sorted by file names please. I.e. this new line should be added after `export /*...*/ from './binding-key'`.